src/sections/Community/Handbook/community.js

```diff
@@ -15,6 +15,7 @@ import NightHawkLogo from "../../../assets/images/nighthawk/icon-only/SVG/nighth
 import TocPagination from "../../../components/handbook-navigation/TocPagination";
 import IntraPage from "../../../components/handbook-navigation/intra-page";
 import uiuxrLogo from "../../../assets/images/uiuxr/uiuxr.svg";
+import writersLogo from "../../../assets/images/writer-program/writer-program-badge.svg";
```

Comment:
@eeshaanSA will you please see that this is moved to an appropriate folder containing other community badges?

Reply:
Actually @leecalcote , the svg file was initially present in the folder. I just had to link it in handbook.